tests/local_cluster.rs

```diff
@@ -109,3 +110,21 @@ fn test_two_unbalanced_stakes() {
 
     drop(cluster);
 }
+
+#[test]
+fn test_forwarding() {
+    // Set up a cluster where one node is never the leader, so all txs sent to this node
+    // will be have to be forwarded
+    let fullnode_config = FullnodeConfig::default();
+    let cluster = LocalCluster::new_with_config(&[999_990, 3], 2_000_000, &fullnode_config);
+
+    let cluster_nodes = discover(&cluster.entry_point_info.gossip, 2).unwrap();
+    assert!(cluster_nodes.len() >= 2);
+
+    let leader_id = cluster.entry_point_info.id;
+
+    let validator_info = cluster_nodes.iter().find(|c| c.id != leader_id).unwrap();
+    cluster_tests::send_many_transactions(&validator_info, &cluster.funding_keypair, 20);
+
```

Comment:
This test doesn't really test anything - there's no assertion. There's a comment saying that it's testing forwarding, but nothing checking that anything was actually forwarded.

Reply:
The send_many_transactions() function sends transactions to a validator node, and then checks that those signatures are confirmed on the validator (the retry_transfer() function does this). The only way those signatures are confirmed is if those transactions were successfully forwarded to the leader